### DISCUSSION TITLE
Welcome to altcampv1-frontend Discussions!

### DISCUSSION THREAD

**Discussion by Boye95** [AltCamp/altcampv1-frontend#227] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Boye95** [comment]
Ideas! Ideas! Ideas!

**Reply by Hackatonic** [reply]
I am Stephen Akinpelu, one of the product managers that worked with Mustapha on the AltCamp product

**Reply by Boye95** [reply]
Hi Stephen. It's good to have you here.  Thinking of writing code?👀
